frontend/src/components/Field/YesNo/YesNo.tsx

```diff
@@ -44,6 +44,11 @@ export interface YesNoProps {
    * Color scheme of the component to render. Defaults to `primary`.
    */
   colorScheme?: FieldColorScheme
+
+  /**
+   * Whether the component is in a high contrast state.
+   */
+  highContrast?: boolean
```

Comment:
```suggestion
  isHighContrast?: boolean
```

recommend for boolean values to follow naming convention eg, `isHighContrast` for better readability.

Reply:
eg, `isOpen` seen in the code